sources/updater/updog/src/error.rs

```diff
@@ -66,6 +66,20 @@ pub(crate) enum Error {
         backtrace: Backtrace,
     },
 
+    #[snafu(display("Manifest load error: {}", source))]
+    ManifestLoad {
+        source: tough::error::Error,
+        backtrace: Backtrace,
```

Comment:
Is there a rationale behind which errors have a backtrace?

Reply:
Not that I'm aware of. I guess it looks like they all do in updog. Should I add it to the new errors? I'm not sure the value of it. I have never noticed a Backtrace in a snafu error print: `{:?}`